src/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ ADD_SUBDIRECTORY(drm)
 ADD_SUBDIRECTORY(dstripe)
 ADD_SUBDIRECTORY(dsync)
 IF(ENABLE_DAOS AND ENABLE_HDF5)
+  ADD_SUBDIRECTORY(daos-gen)
```

Comment:
@dsikich Should we put `daos-gen` under experimental since it's really just being used for testing purposes?

Reply:
@daltonbohning yeah, I suppose my issue while writing the code I have to build all of the experimental tools